pkg/controller/route/routecontroller.go

```diff
@@ -97,12 +107,20 @@ func (rc *RouteController) reconcile(nodes []api.Node, routes []*cloudprovider.R
 			wg.Add(1)
 			glog.Infof("Creating route for node %s %s with hint %s", node.Name, route.DestinationCIDR, nameHint)
 			go func(nodeName string, nameHint string, route *cloudprovider.Route, startTime time.Time) {
-				if err := rc.routes.CreateRoute(rc.clusterName, nameHint, route); err != nil {
-					glog.Errorf("Could not create route %s %s for node %s after %v: %v", nameHint, route.DestinationCIDR, nodeName, time.Now().Sub(startTime), err)
-				} else {
-					glog.Infof("Created route for node %s %s with hint %s after %v", nodeName, route.DestinationCIDR, nameHint, time.Now().Sub(startTime))
+				defer wg.Done()
+				for i := 0; i < maxRetries; i++ {
+					// Ensure that we don't have more than maxConcurrentRouteCreations
+					// CreateRoute calls in flight.
+					rateLimiter <- struct{}{}
```

Comment:
Oops sorry - I missed it from emails.
I will do it in an hour or two (once my kid is in bed).